### ISSUE TITLE
When the rows are formatted as a line, the comma is not removed

### ISSUE THREAD

**Issue by huhaifan** [ThisIsManta/stylus-supremacy#11]
```
src: url('../fonts/sell-icon.eot?8trqpw#iefix'),
    url('../fonts/sell-icon.ttf?8trqpw') format('truetype'),
    url('../fonts/sell-icon.woff?8trqpw') format('woff'),
    url('../fonts/sell-icon.svg?8trqpw#sell-icon') format('svg');
```
After formatting
```
src: url('../fonts/sell-icon.eot?8trqpw#iefix'), url('../fonts/sell-icon.ttf?8trqpw'), format('truetype'), url('../fonts/sell-icon.woff?8trqpw'), format('woff'), url('../fonts/sell-icon.svg?8trqpw#sell-icon') format('svg')
```

**Comment by ThisIsManta** [comment]
Hi,

Are you expecting the commas to be gone after formatting?

**Comment by huhaifan** [comment]
I think it should be the same after formatting
```
src: url('../fonts/sell-icon.eot?8trqpw#iefix'), url('../fonts/sell-icon.ttf?8trqpw') format('truetype'), url('../fonts/sell-icon.woff?8trqpw') format('woff'), url('../fonts/sell-icon.svg?8trqpw#sell-icon') format('svg')
```

**Comment by ThisIsManta** [comment]
Sorry, I don't quite follow you.

The only difference I can see here is a semicolon.
`,` this is a comma.
`;` this is a semicolon.
Would you tell me if I'm missing something?

**Comment by huhaifan** [comment]
Sorry,my expression is not clear.
The difference is as follows:
```
url('../fonts/sell-icon.ttf?8trqpw') format('truetype')
```
```
url('../fonts/sell-icon.ttf?8trqpw')， format('truetype')
```
The comma at the end of each line should be deleted after formatting

**Comment by ThisIsManta** [comment]
If I'm not mistaken, omitting the commas between `url() format(), url() format()` here would make the compiled CSS **not to work**.

The following HTML would not work, if you removed the comma between `format('woff2'), url(...`
So why would you remove the commas?

```html
<html>
  <head>
    <title></title>

    <style type="text/css" media="screen, print">
      @font-face {
        font-family: 'ABC';
        src: url('https://fonts.gstatic.com/s/kanit/v3/RPQGAQk3DD66RypwKRnBKQ.woff2') format('woff2'), url('https://fonts.gstatic.com/s/rozhaone/v4/UYFFPwb2ZMrchIuzxLAWb1tXRa8TVwTICgirnJhmVJw.woff2') format('woff2');
      }

      body {
        font-family: "ABC", serif
      }
    </style>
  </head>
  <body>
    Test
  </body>
</html>
```

**Comment by huhaifan** [comment]
I just started learning web knowledge, maybe I understand it wrong.😅
I am using this font file in my project.And I want to convert the css file to the styl file,but it does not work after formatting. I found that there was an extra comma after formatting, maybe you could try it yourself.
url('fonts/sell-icon.eot?8trqpw#iefix') format('embedded-opentype'), url('fonts/sell-icon.ttf?8trqpw'), format('truetype'),
The red comma is superfluous.
________________________________
发件人: Anantachai Saothong <notifications@github.com>
发送时间: 2017年10月31日 16:38
收件人: ThisIsManta/stylus-supremacy
抄送: huhaifan; Author
主题: Re: [ThisIsManta/stylus-supremacy] When the rows are formatted as a line, the comma is not removed (#11)


If I'm not mistaken, omitting the commas between url() format(), url() format() here would make the compiled CSS not to work.

The following HTML would not work, if you removed the comma between format('woff2'), url(...
So why would you remove the commas?

<html>
  <head>
    <title></title>

    <style type="text/css" media="screen, print">
      @font-face {
        font-family: 'ABC';
        src: url('https://fonts.gstatic.com/s/kanit/v3/RPQGAQk3DD66RypwKRnBKQ.woff2') format('woff2'), url('https://fonts.gstatic.com/s/rozhaone/v4/UYFFPwb2ZMrchIuzxLAWb1tXRa8TVwTICgirnJhmVJw.woff2') format('woff2');
      }

      body {
        font-family: "ABC", serif
      }
    </style>
  </head>
  <body>
    Test
  </body>
</html>

**Comment by ThisIsManta** [comment]
Ah! I got you now. Please spare me a few days for the fix.

**Comment by huhaifan** [comment]
😂
________________________________
发件人: Anantachai Saothong <notifications@github.com>
发送时间: 2017年11月1日 2:08
收件人: ThisIsManta/stylus-supremacy
抄送: huhaifan; Author
主题: Re: [ThisIsManta/stylus-supremacy] When the rows are formatted as a line, the comma is not removed (#11)


Ah! I got you now. Please spare me a few days for the fix.

**Comment by ThisIsManta** [comment]
This is fixed in [version 2.5.2](https://github.com/ThisIsManta/stylus-supremacy/releases/tag/v2.5.2). You can update NPM module and/or VSCode extension right away.
Cheers!
